### ISSUE TITLE
Feature Request: DNS Probe (Dig)

### ISSUE THREAD

**Issue by derekpurdy** [linuxserver/docker-smokeping#41]
Would it be possible to add dig into the container for DNS tests?

+DNS
binary = /usr/bin/dig

**Comment by thelamer** [comment]
bind-tools provides dig in Alpine, run a test locally and ensure DNS probe works for you. 

Open a pull request and add bind-tools to the DockerFile somewhere in the base package installation here: 

https://github.com/linuxserver/docker-smokeping/blob/master/Dockerfile#L15

IE: 

```
 apk add --no-cache \
	apache2 \
	apache2-ctl \
	apache2-utils \
        bind-tools \
```

**Comment by IronicBadger** [comment]
Closed by #39
